client/src/FormFrontend/FormFrontend.js

```diff
@@ -297,6 +297,39 @@ class FormFrontend extends React.Component {
             </Fragment>
           );
         }
+        case "form_faq":
+          return (
+            <>
+              <div style={{ fontWeight: 700 }}>
+                {text_maker("form_frontend_faq_note")}
+                {required_asterisk}
+              </div>
+              <FancyUL>
+                {_.map(field_info.faq_content, ({ q, a }) => {
+                  return (
+                    <div key={_.uniqueId("form-faq-qa-pair_")}>
```

Comment:
`key`s should be stable across renders, see the docs here for [more](https://reactjs.org/docs/lists-and-keys.html#keys) on that.

In cases like this where the components in the returned array will not be changing across renders and will always be in the same order, using the index argument from the `map` for the `key` is totally good enough :+1: